<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
	background-color: rgb(255, 255, 255);}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_pendente">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: rgb(33, 66, 166);
border-bottom-color: rgb(0, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pendente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>610</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Desenvolvido por @phporath</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;; color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="apply_analise">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>540</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: rgb(33, 66, 166);
border-bottom-color: rgb(0, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Em análise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_concluido">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>540</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: rgb(33, 66, 166);
border-bottom-color: rgb(0, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Concluído</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bt_SV">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Sistema Viário</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="bt_IP">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Iluminação Pública</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: rgb(33, 66, 166);
border-bottom-color: rgb(0, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
    <property name="icon">
     <iconset theme="back"/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
